csharp-advanced-topics/DelegatesInCsharp/ActionFuncTests/OtherTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Linq;
+
+namespace ActionFunc.Tests
+{
+    [TestClass()]
+    public class OtherTests
+    {
+        [TestMethod()]
+        public void AnonymousFuncTest()
+        {
+            try
+            {
+                int a = 3;
+                int b = 9;
+                string expected = "27";
+                Func<int, int, string> myAnonimousFunc = (a, b) => { return $"{a * b}"; };
+                var result = myAnonimousFunc(a, b);
+                Assert.AreEqual(expected, result);
+            }
+            catch
+            { Assert.IsTrue(false); }
+        }
+
+        [TestMethod()]
+        public void AnonymousActionTest()
```

Comment:
Test method names are not in accordance with the naming conventions we've mentioned in the guidelines. Please change them.